pyomeca/processing/interp.py

```diff
@@ -0,0 +1,22 @@
+from typing import Union
+
+import numpy as np
+import xarray as xr
+
+
+def time_normalize(
+    array: xr.DataArray,
+    time_vector: Union[xr.DataArray, np.array] = None,
+    n_frames: int = 100,
+    norm_time: bool = False,
+) -> xr.DataArray:
+    if time_vector is None:
+        if norm_time:
+            first_last_time = (0, 99)
```

Comment:
I feel that the default values should be 0-100, with a total of 101 values

Reply:
It always bothered me when `spm1d` was doing that because if you normalize, it's usually to talk in % of the movement and if you do that you can get a 101%?

Reply:
I do not think so. For me the first value is 0% (and not 1%) and the last value is 100% (not 99%). If you run numpy with `np.linspace(0, 100, 101)` you get [0, 1, 2, ..., 98, 99, 100] as expected. 
For me the last value should be 100% and not 99% (as it is if last time is 0-99)